### PR TITLE
[9.0][UPD] Several renames and merges

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
This PR adds several renames and merges.

### CODE REVIEW

File: openerp/addons/openupgrade_records/lib/apriori.py [hunk 1]
```diff
@@ -46,6 +49,11 @@
     'hr_expense_analytic_plans': 'hr_expense_analytic_distribution',
     # OCA/project
     'project_work_time_control': 'project_timesheet_time_control',
+    # OCA/manufacture-reporting
+    'mrp_bom_structure_xls': 'mrp_bom_structure_xlsx',
+    'mrp_bom_structure_xls_level_1': 'mrp_bom_structure_xlsx_level_1',
+    # OCA/reporting-engine
+    'report_xls': 'report_xlsx',
```
Comment (pedrobaeza) [inline openerp/addons/openupgrade_records/lib/apriori.py:56]: This one can't be renamed this way: you can be using report_xls for other modules that haven't been converted to XLSX.

File: openerp/addons/base/migrations/9.0.1.3/pre-migration.py [hunk 2]
```diff
@@ -59,6 +59,7 @@ def cleanup_modules(cr):
         cr, [
             ('contacts', 'mail'),
             ('marketing_crm', 'crm'),
+            ('email_template', 'mail'),  # mail_template class
```
Comment (pedrobaeza) [inline openerp/addons/base/migrations/9.0.1.3/pre-migration.py:62]: This one is already done in https://github.com/OCA/OpenUpgrade/blob/9.0/addons/mail/migrations/9.0.1.0/pre-migration.py#L25. Maybe it's better to move it here, but remove it from the other place then.

File: openerp/addons/openupgrade_records/lib/apriori.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply openerp/addons/openupgrade_records/lib/apriori.py:56]: I disagree. `report_xls` of `reporting-engine` is not installable in v9. We are doing thinks based on what OCA has now in the repositories, not based on what some individuals may do.

Comment (pedrobaeza) [reply openerp/addons/openupgrade_records/lib/apriori.py:56]: OK, I thought it was ported (you did the PR), then let's remove it definitively report_xls in OCA in both v9 and v10.

File: openerp/addons/base/migrations/9.0.1.3/pre-migration.py (hunk 2 above)
Comment (MiquelRForgeFlow) [reply openerp/addons/base/migrations/9.0.1.3/pre-migration.py:62]: Done

File: openerp/addons/openupgrade_records/lib/apriori.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply openerp/addons/openupgrade_records/lib/apriori.py:56]: Yeah, could you remove them?

Comment (pedrobaeza) [reply openerp/addons/openupgrade_records/lib/apriori.py:56]: Well, I was thinking more in you for that task :stuck_out_tongue_closed_eyes:

Comment (MiquelRForgeFlow) [reply openerp/addons/openupgrade_records/lib/apriori.py:56]: 😒 ok..., I will do it now 😅

Comment (pedrobaeza) [reply openerp/addons/openupgrade_records/lib/apriori.py:56]: Thanks! I will be in the review side!

File: openerp/addons/base/migrations/9.0.1.3/pre-migration.py (hunk 2 above)
Comment (MiquelRForgeFlow) [reply openerp/addons/base/migrations/9.0.1.3/pre-migration.py:62]: Should I move the associated model and table renames scripts?

Comment (pedrobaeza) [reply openerp/addons/base/migrations/9.0.1.3/pre-migration.py:62]: No, that should be in the corresponding module. This is only the initiation of the migration, for setting which modules to upgrade. When upgrading the mail module is when renamings should happen.

Comment (MiquelRForgeFlow) [reply openerp/addons/base/migrations/9.0.1.3/pre-migration.py:62]: Ok, fine!

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**